setup.py

```diff
@@ -22,6 +22,8 @@
     'numpy>=1.19.5',
     'pandas>=1.2.0',
     'scikit-learn>=0.24.1',
+    'xgboost==1.4.2',
+    'lightgbm==3.2.1'
```

Comment:
These are not really requirements of the package. I use them for the tests just to be sure they won't break but the functionalities of the package do not depend on them. 

So probably they belong to the test_require